app/scripts/modules/core/delivery/executionGroup/execution/ExecutionMarker.tsx

```diff
@@ -45,28 +50,40 @@ export class ExecutionMarker extends React.Component<IExecutionMarkerProps, IExe
   private handleStageClick = () => {
```

Comment:
If we merge @christopherthielen PR with the `@autobindMethods` decorator first, this should be updated.